src/main/java/ca/corefacility/bioinformatics/irida/ria/web/services/UIProjectMembersService.java

```diff
@@ -0,0 +1,138 @@
+import ca.corefacility.bioinformatics.irida.model.enums.ProjectRole;
+import ca.corefacility.bioinformatics.irida.model.joins.Join;
+import ca.corefacility.bioinformatics.irida.model.joins.impl.ProjectUserJoin;
+import ca.corefacility.bioinformatics.irida.model.project.Project;
+import ca.corefacility.bioinformatics.irida.model.user.User;
+import ca.corefacility.bioinformatics.irida.ria.web.exceptions.UIProjectWithoutOwnerException;
+import ca.corefacility.bioinformatics.irida.ria.web.models.tables.TableRequest;
+import ca.corefacility.bioinformatics.irida.ria.web.models.tables.TableResponse;
+import ca.corefacility.bioinformatics.irida.ria.web.projects.dto.ProjectMemberTableModel;
+import ca.corefacility.bioinformatics.irida.ria.web.projects.settings.dto.NewProjectMemberRequest;
+import ca.corefacility.bioinformatics.irida.service.ProjectService;
+import ca.corefacility.bioinformatics.irida.service.user.UserService;
+
+/**
+ * Service class for the UI for handling project members actions.
+ */
+@Component
+public class UIProjectMembersService {
+	private final ProjectService projectService;
+	private final UserService userService;
+	private final MessageSource messageSource;
+
+	@Autowired
+	public UIProjectMembersService(ProjectService projectService, UserService userService, MessageSource messageSource) {
+		this.projectService = projectService;
+		this.userService = userService;
+		this.messageSource = messageSource;
+	}
+
+	/**
+	 * Get a paged listing of project members passed on parameters set in the table request.
+	 *
+	 * @param projectId    - identifier for the current project
+	 * @param tableRequest - details about the current page of the table
+	 * @return sorted and filtered list of project members
+	 */
+	public TableResponse<ProjectMemberTableModel> getProjectMembers(Long projectId, TableRequest tableRequest) {
+		Project project = projectService.read(projectId);
+		Page<Join<Project, User>> usersForProject = userService.searchUsersForProject(project, tableRequest.getSearch(),
+				tableRequest.getCurrent(), tableRequest.getPageSize(), tableRequest.getSort());
+		List<ProjectMemberTableModel> members = usersForProject.get()
+				.map(join -> {
+					ProjectUserJoin projectUserJoin = (ProjectUserJoin) join;
+					return new ProjectMemberTableModel(join.getObject(), projectUserJoin.getProjectRole()
+							.toString(), projectUserJoin.getCreatedDate());
+				})
+				.collect(Collectors.toList());
+		return new TableResponse<>(members, usersForProject.getTotalElements());
+	}
+
+	/**
+	 * Remove a user from the project
+	 *
+	 * @param projectId - identifier for the current project
+	 * @param userId    - identifier for the user to remove from the project
+	 * @param locale    - of the currently logged in user
+	 * @return Message to display to the user about the outcome of removing a user from the project.
+	 * @throws UIProjectWithoutOwnerException if removing the user will leave the project without a manage
```

Comment:
Change `manage` to `manager`

Reply:
Fixed in 5b4a37e24b443d99df6ef574ff99fe5368db3ebb

Reply:
Thanks